requirements/edx/custom.txt

```diff
@@ -15,4 +15,5 @@
 -e git+https://github.com/open-craft/xblock-group-project-v2.git@90fb6f8db6ed838d46460c153c7ee79e71dd08d3#egg=xblock-group-project-v2
 -e git+https://github.com/OfficeDev/xblock-officemix.git@86238f5968a08db005717dbddc346808f1ed3716#egg=xblock-officemix
 -e git+https://github.com/edx-solutions/xblock.git@d25a661454da9e6b149d6c559de50ddcd0770857#egg=xblock
+-e git+https://github.com/edx-solutions/organizations-edx-platform-extensions.git@ffa82a0f2e5a72b1924a08ed8da9b942bca627fc#egg=organizations-edx-platform-extensions
```

Comment:
It would be better to configure this as a non-editable install, if possible.  In addition we should use release tags for these new references instead of git hashes.